### PR TITLE
add last updated field to SampleLineages

### PR DESCRIPTION
**Pull request by phoenixAja**
### Summary:
- **What:** add last_updated field to SampleLineages table
- **Ticket:** [sc<fill_in_issue_number>](https://app.shortcut.com/genepi/story/<fill_in_issue_number>)
- **Env:** `<rdev link>`

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

**Review by jgadling — COMMENTED** [review]
Does this PR update the pangolin job to populate the new field?

**Comment by phoenixAja** [comment]
@jgadling no the next PR did where we removed the pangolin fields from the pathogen_genome class, but let me add that now to this PR

**Review by lvreynoso — APPROVED** [review]
just lint before merging (`make backend-run-style`)

File: src/backend/aspen/api/schemas/samples.py [hunk 1]
```diff
@@ -24,11 +24,12 @@ class SampleGisaidResponse(BaseResponse):
     status: str
 
 
+# TODO: add field to capture pangolin_last_updated (last_updated) once we add support for that
```
Comment (lvreynoso) [inline src/backend/aspen/api/schemas/samples.py]: is this not what this PR is?

Comment (phoenixAja) [reply src/backend/aspen/api/schemas/samples.py]: yes thanks for catching! haha

**Pull request merged**